### PR TITLE
V15 0

### PR DESCRIPTION
**Pull request by ben-kaufman**
Improved DAO script

### CODE REVIEW

File: migrate-dao.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const utils = require('./utils.js')
 async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logTx, previousMigration: { base } }) {
-  opts.gas = 6721975
+  opts.gas = 8000000
```
Comment (orenyodfat) [inline migrate-dao.js]: lets define that somewhere so the same value can used by other script.

File: migrate-dao.js [hunk 2]
```diff
@@ -90,26 +90,50 @@ async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logT
...
+      tokenDist.slice(0, foundersInitCount),
+      repDist.slice(0, foundersInitCount),
       migrationParams.useUController === true ? UController : '0x0000000000000000000000000000000000000000',
       '0'
     )
 
-    const Avatar = await forgeOrg.call()
-    tx = await forgeOrg.send()
+    tx = await forgeOrg.send({
+      'gas': 6500000
```
Comment (orenyodfat) [inline migrate-dao.js]: how about `opt.gas `?

File: migrate-dao.js [hunk 3]
```diff
@@ -90,26 +90,50 @@ async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logT
...
+      i++
+      spinner.start('Adding founders...')
+      let currentBatchCount = foundersToAddCount < foundersBatchSize ? foundersToAddCount : foundersBatchSize
+      tx = await daoCreator.methods.addFounders(
+        Avatar,
+        founderAddresses.slice(i * foundersBatchSize, i * foundersBatchSize + currentBatchCount),
+        tokenDist.slice(i * foundersBatchSize, i * foundersBatchSize + currentBatchCount),
+        repDist.slice(i * foundersBatchSize, i * foundersBatchSize + currentBatchCount)
+      ).send({
+        'gas': 6500000
```
Comment (orenyodfat) [inline migrate-dao.js]: how about opt.gas ?

Comment (ben-kaufman) [reply migrate-dao.js]: I'm not sure if that's really necessary actually. I think I might just add it when trying to fix something but I see we already set a reasonable gas in the migrate.js script. I'll just remove it for now.

**Pull request merged**